client/src/components/WelcomePage.js

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from "react";
+import './welcomepagestyle.css'
+import adminpic from './admin.png'
+
+
+function WelcomePage() {
+
+  window.onscroll = function () { myFunction() };
+
+  var sty1 = {
+    FontColor: "black",
+    opacity: "0.8",
+
+
+  }
+  var sty2 = {
+    position: "fixed",
+    width: "100%",
+    backgroundColor: "black"
+
+
+  }
+  const [sty, setsty] = useState(sty1);
+
+
+
+
+  function myFunction() {
+    if (window.pageYOffset === 0)
+      setsty(sty1);
+    else
+      setsty(sty2);
+  }
+
+  return (
+
+    <div>
+      <div id="navbar" style={sty}>
+        <a href="#home" class="tx" >?Kurakoo</a>
```

Comment:
Hey @ASHISH-GITHUB2495 , great work! But why did you use "class" everywhere instead of "className", use of "class" would generate unnecessary console errors.

Reply:
Brother , if they are unnecessary then why you bother it ... actually it was just flow of mine and at the time of completion of this whole project ... I will generate an issue to solve all the console errors ... Thank you for evaluation.